testsuite/tests/apicast/parameters/http_proxy/test_http_proxy.py

```diff
@@ -22,7 +22,7 @@ def gateway_kind():
 @pytest.fixture(scope="module")
 def service_proxy_settings(private_base_url):
     "Dict of proxy settings to be used when service created"
-    return rawobj.Proxy(private_base_url("httpbin_service"))
+    return rawobj.Proxy(private_base_url("go-httpbin"))
```

Comment:
shouldnt be there `"echo-api+svc:9292"`

Reply:
No it should not. a) This is part of effort to eliminate dependency on echo-api. b) this PR is screwed little bit, please look at commit message directly for explanation